resistance/resistance.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program calculates the resistance R of two resistors.")
+	fmt.Println("Enter the resistance of two resistors:")
+	var a, b float64
+	fmt.Scanln(&a, &b)
+	R := a + b
+	R2 := 1/a + 1/b
```

Comment:
technically, this isn't resistance, it's a conductance (1/resistance). it should be named `G`, not `R2`